db/migrate/20150416085800_associate_parent_policies.rb

```diff
@@ -0,0 +1,16 @@
+class AssociateParentPolicies < ActiveRecord::Migration
+  POLICY_PARENT_MAPPINGS = {
+    'automatic-enrolment-in-workplace-pensions' => ['employment', 'older-people'],
+    'city-deal' => ['localism'],
+    'regional-growth-fund' => ['employment'],
+    'hs2-high-speed-rail' => ['rail-network']
+  }
+
+  def change
+    POLICY_PARENT_MAPPINGS.each do |policy_slug, parent_policy_slugs|
+      policy = Policy.find_by!(slug: policy_slug)
+      parents = Policy.where(slug: parent_policy_slugs)
+      policy.parent_policies = parents
```

Comment:
It's not necessary for the purposes of saving the association as the join records are created with the assignment.